presto-memory/src/main/java/com/facebook/presto/plugin/memory/MemoryMetadata.java

```diff
@@ -206,6 +211,10 @@ private void checkTableNotExists(SchemaTableName tableName)
                 .anyMatch(tableName::equals)) {
             throw new PrestoException(ALREADY_EXISTS, format("Table [%s] already exists", tableName.toString()));
         }
+        if (views.keySet().stream()
```

Comment:
Nit: I wouldn't wrap here, since it's short and there's only one item, so wrapping makes it harder to read

Reply:
Actually, I think this can just be `views.keySet().contains(tableName)`